src/modules/dashboard/components/ActionsPage/InputStorageWidget/InputStorageWidget.css

```diff
@@ -0,0 +1,49 @@
+.widgetForm {
+  margin-top: 20px;
+  margin-bottom: 55px;
+  position: relative;
+}
+
+.widgetForm label {
+  line-height: 18px;
+  color: var(--dark);
+}
+
+.learnMoreLink {
+  position: absolute;
+  top: 0;
+  right: 0;
+  z-index: 1;
+  font-size: var(--size-small);
+  font-weight: var(--weight-bold);
+  color: var(--action-secondary);
+}
+
+.currentValueText {
+  margin-bottom: 2px;
+  font-size: var(--size-tiny);
+  font-weight: var(--weight-bold);
+  line-height: 18px;
+}
+
+.formFooter {
+  text-align: right;
+}
```

Comment:
The top margin for the footer section is still too big.

Yes, I know that's coming actually from the input component.

To fix, this, just set this element's top margin to a negative value.

I've tested and `-19px` seems to be working fine.